web/index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>BCU Labs</title>
+    <meta name="viewport"
+          content="initial-scale=1,maximum-scale=1,user-scalable=no">
+    <link href="https://api.mapbox.com/mapbox-gl-js/v3.1.2/mapbox-gl.css"
+          rel="stylesheet">
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+
+        #map {
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            width: 100%;
+        }
+    </style>
+</head>
+<body>
+<h1>BCU Data Labs</h1>
+<div id="map"></div>
+<script src="https://api.mapbox.com/mapbox-gl-js/v3.1.2/mapbox-gl.js"></script>
+<script>
+    const baseWidth = 1.4;
+    const baseZoom = 12
+
+    mapboxgl.accessToken = 'pk.eyJ1IjoidHNjaGllZ2dtIiwiYSI6ImNrZHoyb25iYjMxMWQzM2p6eHlibHJkanIifQ.wj-SisFvNPgakxBy_1ZnHw';
+    const map = new mapboxgl.Map({
+        container: 'map', // container ID
+        center: [-71.057083, 42.361145], // Boston starting position [lng, lat]
+        style: "mapbox://styles/tschieggm/cke3eofps022p19mqmvfc8n4q",
```

Comment:
Is this possible to be defined locally? It'd be nice to be able to play with the style and try things out.

Reply:
I don't think it can be defined locally, but this could be a config value that you define when you run the web server. I can pull it out into config that's detected at startup.

Reply:
I'm actually going to write a TODO to have it be populated at server startup later when we consider moving to Flask+Jinja